sensor_control/vision/symbols/src/FrameProc.h

```diff
@@ -0,0 +1,51 @@
+//Grabs frame from ROS, erode/dilate, converts to HSV, produces Red Green Blue Black binary frames;
+#ifndef FROM_PROC_H
```

Comment:
`#pragma once` does the same thing as these include guards but with much less typing / worrying about unique names.